requirements-test.txt

```diff
@@ -1,2 +1,2 @@
 docker~=7.1.0
-openwisp-utils[qa,selenium]~=1.1.1
+openwisp-utils[qa,selenium] @ https://github.com/openwisp/openwisp-utils/tarball/1.2
```

Comment:
After merging the PR on openwisp-utils and updating this line the build failed, maybe the github action reference needs to be updated too?